data/json/traps.json

```diff
@@ -573,6 +573,21 @@
         "drops" : ["leather_funnel"],
         "benign" : true,
         "funnel_radius": 85
+    },{
+        "type" : "trap",
+        "id": "tr_funnel",
```

Comment:
This means it would override the existing `tr_funnel` trap, it should be a new id.